Description: Adapt to API changes
 This adapts to API changes in qtermwidget which were made when fixing a
 memory problem when right-clicking on links.
Author: Yen Chi Hsuan <yan12125@gmail.com>
Origin: upstream
Bug: https://github.com/lxqt/qterminal/issues/358
Applied-Upstream: commit:cbdb9e2
Last-Update: 2018-07-10
--- a/src/termwidget.cpp
+++ b/src/termwidget.cpp
@@ -137,7 +137,7 @@ void TermWidgetImpl::customContextMenuCa
     QMenu menu;
     QMap<QString, QAction*> actions = findParent<MainWindow>(this)->leaseActions();
 
-    QList<QAction*> extraActions = filterActions(pos);
+    QList<QAction*> extraActions = filterActions(pos, &menu);
     for (auto& action : extraActions)
     {
         menu.addAction(action);
